build_all_images.sh

```diff
@@ -593,6 +620,8 @@ function main {
 	#build_zabbix_server_image
 	#build_zabbix_web_image
 
+	#build_bundler_builder_image
```

Comment:
@Akos-Kreutz, can we sort like [this](https://github.com/Akos-Kreutz/liferay-docker/pull/14/commits/4cc7d37e37e2a2feff5a40ff0c2d87099ed8dae7)?

Reply:
@natocesarrego Done